locale/en/Pacifist.cfg

```diff
@@ -1,3 +1,18 @@
+[gui]
+guns=
+weapons=
+ammo=
+
+[inventory-full-message]
+guns=
```

Comment:
Same as above, I'd rather have a different message than none. 
How about "Tool" and "Tool ammo" or "Tool charges"? See also tooltips for ammo:
![image](https://github.com/Derpumu/Pacifist/assets/183507/526f9ceb-ce14-4d43-a1f7-dbe715e7be1b)

Reply:
I had missed that one. Now that "Ammo" says "Charges".